app/test/unit/git/cherry-pick-test.ts

```diff
@@ -264,4 +271,126 @@ describe('git/cherry-pick', () => {
     }
     expect(result).toBe(null)
   })
+
+  describe('cherry picking with conflicts', () => {
+    beforeEach(async () => {
+      // In the 'git/cherry-pick' `beforeEach`, we call `createRepository` which
+      // adds a commit to the feature branch with a file called THING.md. In
+      // order to make a conflict, we will add the same file to the target
+      // branch.
+      const conflictingCommit = {
+        commitMessage: 'Conflicting Commit!',
+        entries: [
+          {
+            path: 'THING.md',
+            contents: '# HELLO WORLD! \n CREATING CONFLICT! FUN TIMES!\n',
+          },
+        ],
+      }
+      await makeCommit(repository, conflictingCommit)
+    })
+
+    it('successfully detect cherry pick with conflicts', async () => {
```

Comment:
Nit: This is like the final boss of nits, I'm super super sorry 😳   But in theory these verbs should be in 3rd person because `it` is supposed to be the subject in the sentence:
```suggestion
    it('successfully detects cherry pick with conflicts', async () => {
```

The same applies to the rest of test cases.

Feel free to ignore my comment for this PR, but I think it'd be nice to follow that convention like we do in other test cases 😳

Reply:
Wow.. I don't think I have seen anyone pose it like that... but it just makes perfect sense. 😂

Reply:
Done!